e2e/tests/components/column-resizing/column-resizing-sorting/column-resizing-sorting.po.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { $, browser, ElementFinder } from 'protractor';
+
+export class ColumnResizingSortingPage {
+
+    table = $('table');
+    nameHeader = $('th');
+
+    async getPage(): Promise<void> {
+        await browser.get('#/column-resizing/column-resizing-sorting');
+    }
+
+    async resizeColumn(columnIndex: number, amount: number): Promise<void> {
+        const columns = await this.getColumnHeaders();
+        const column = columns[columnIndex];
+        const handle = await column.$('.ux-resizable-table-column-handle');
```

Comment:
I think `getColumnHeaders` could become `getColumnResizeHandle` instead with the above code, which would be more useful.